docs/history/changelog.md

```diff
@@ -160,6 +160,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
     * `contracts.maximumValue`
     * `contracts.estimatedValue`
   * [#1372](https://github.com/open-contracting/standard/pull/1372) `Address.country`
+  * [#1421](https://github.com/open-contracting/standard/pull/1421) `tender.standstillPeriod`
```

Comment:
Move under `tender.datePublished` to keep this in schema order.

Reply:
OK - I've made a PR to update the style guide https://github.com/open-contracting/standard-development-handbook/pull/252

Reply:
Done in https://github.com/open-contracting/standard/pull/1421/commits/a2fde4a3ba5926105daee2d46e2264d394ba6267